instrumentation/mysql6/src/test/java/brave/mysql6/ITTracingStatementInterceptor.java

```diff
@@ -0,0 +1,141 @@
+package brave.mysql6;
+
+import brave.Tracer.SpanInScope;
+import brave.Tracing;
+import brave.internal.StrictCurrentTraceContext;
+import brave.sampler.Sampler;
+import java.sql.Connection;
+import java.sql.PreparedStatement;
+import java.sql.ResultSet;
+import java.sql.SQLException;
+import java.util.concurrent.ConcurrentLinkedDeque;
+
+import com.mysql.cj.jdbc.MysqlDataSource;
+import org.junit.After;
+import org.junit.Before;
+import org.junit.Test;
+import zipkin.Constants;
+import zipkin.Span;
+import zipkin.TraceKeys;
+import zipkin.internal.Util;
+
+import static org.assertj.core.api.Assertions.assertThat;
+import static org.assertj.core.api.Assertions.tuple;
+import static org.junit.Assume.assumeTrue;
+import static zipkin.internal.Util.envOr;
+
+@SuppressWarnings("Duplicates")
+public class ITTracingStatementInterceptor {
+  static final String QUERY = "select 'hello world'";
+
+  ConcurrentLinkedDeque<Span> spans = new ConcurrentLinkedDeque<>();
+
+  Tracing tracing = tracingBuilder(Sampler.ALWAYS_SAMPLE).build();
+  Connection connection;
+
+  @Before public void init() throws SQLException {
+    StringBuilder url = new StringBuilder("jdbc:mysql://");
+    url.append(envOr("MYSQL_HOST", "127.0.0.1"));
+    url.append(":").append(envOr("MYSQL_TCP_PORT", 3306));
+    String db = envOr("MYSQL_DB", null);
+    if (db != null) url.append("/").append(db);
+    url.append("?statementInterceptors=").append(TracingStatementInterceptor.class.getName());
+    url.append("&zipkinServiceName=").append("myservice");
+
+    MysqlDataSource dataSource = new MysqlDataSource();
+    dataSource.setUrl(url.toString());
+
+    dataSource.setUser(System.getenv("MYSQL_USER"));
```

Comment:
ps did you try this? I think CI will (ex `MYSQL_USER=root ./mvnw clean install`)